app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(car.comfort_class
+                     * (self.clean_power - car.clean_mark)
+                     * self.average_rating
+                     / self.distance_from_city_center, 1
+                     )
+
+    def serve_cars(self, car_ls: Car) -> float:
+        income = 0
+        for car in car_ls:
+            if self.clean_power >= car.clean_mark:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return income
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def rate_service(self, num: int) -> float:
+        new_rating = round(((self.average_rating
+                             * self.count_of_ratings) + num)
+                           / (self.count_of_ratings + 1), 1)
+        self.average_rating = new_rating
+        self.count_of_ratings += 1
+        return self.average_rating
```

Comment:
I think you should not return anything in this method, because you are already changing the attribute